pkg/smi/types.go

```diff
@@ -70,11 +70,14 @@ type MeshSpec interface {
 	// ListTrafficTargets lists TrafficTarget SMI resources.
 	ListTrafficTargets() []*target.TrafficTarget
 
-	// ListBackpressures lists Backpressure CRD resources.
+	// ListBackpressures lists Backpressure resources.
 	// This is an experimental feature, which will eventually
 	// in some shape or form make its way into SMI Spec.
 	ListBackpressures() []*backpressure.Backpressure
 
+	// GetBackpressurePolicy gets the Backpressure policy corresponding to the MeshService
+	GetBackpressurePolicy(service.MeshService) *backpressure.Backpressure
```

Comment:
You could theoretically have more than 1 of these backpressure policies match a label. Should we make this a list as well?

Reply:
Which is why I don't think we should be using label selector for the service. The CRD should be explicit about the target service instead.